submissions/veronkrot/styles.css

```diff
@@ -0,0 +1,129 @@
+body {
+    font-family: Roboto, sans-serif;
+    font-size: 16px;
+}
+
+a {
+    color: #4a4a4a;
+    text-decoration: none;
+}
+
+.header {
+    display: flex;
+    justify-content: flex-end;
+}
+
+.nav {
+    display: flex;
+    justify-content: space-between;
+    width: 260px;
+    margin: 20px;
+}
+
+.nav-link {
+    margin-top: 2px;
+}
+
+.bell-icon {
+    width: 20px;
+    margin-top: 3px;
+}
+
+.account-icon {
+    width: 32px;
+    margin-top: -4px;
+}
+
+.apps-button,
+.list-item,
+.more-apps-button {
+    cursor: pointer;
+}
+
+.apps-checkbox,
+.more-apps-checkbox,
+.disabled-item,
+.dividing-line,
+.more-from-kottans,
+.more-apps-checkbox:checked ~ .more-apps-button {
+    display: none;
```

Comment:
Try to hide checkbox in right way. [Video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s)

Reply:
fixed